WPMedia/fusion-news-theme-blocks
# PEN-1381: Footer Icons and Results Read More button should use primary color

## Description
Updated footer so the social icons are rendered using the configured primary color.
Updated results lists so that its read more button is rendered using the configured primary color.

## Jira Ticket
- [PEN-1381](https://arcpublishing.atlassian.net/browse/PEN-1381)

## Acceptance Criteria

The See More button (on both the Results List) and the Footer Icons do not use the customer's primary color Theme Setting. Instead, it seems to be hard-coded to use a blue color: https://corecomponents.arcpublishing.com/pf/all-blocks/?_website=dagen
This button should inherit the website's primary color, which in this case (for the Dagen website) is a red – like some other blocks already do correctly https://github.com/WPMedia/Fusion-News-Theme/blob/master/blocks.json#L207


## Test Steps

1. Checkout branch
2. data restore from core components production pb data tile
3. npm fusion start
4. visit the local page:      http://localhost/pf/all-blocks/?_website=dagen
5. confirm that the Results List block is rendered useing the configured primary color (for Dagen it is a dark red)
6. Confirm that the Footer social icons and border are rendered using the configured primary color ( for Dagen it is a dark red color)

## Effect Of Changes
### Before
Results list Read More button is hard coded to be blue.

Footer social icons and border are hard coded to be blue.

### After
Results list Read More button uses the configured primary color.

Footer social icons and border use configured primary color.


## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json- NONE
- Changes to the custom fields which will require users to reconfigure features - NONE
- Update to css framework or SDK - NONE
- Dependency on another PR that needs to be merged first - NONE

## Review Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [ x] Confirmed all the test steps above are working
- [x ] Confirmed this PR has reasonable code coverage
  - [x ] Confirmed this PR has unit test files
  - [ x] Ran `npm test`, made sure all tests are passing
  - [x ] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [x ] Confirmed relevant documentation has been updated/added.
